EIPS/eip-1.md

```diff
@@ -1,101 +1,100 @@
 
-An EIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+An EEP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
 
-## EIP Work Flow
+## EEP Work Flow
 
-Parties involved in the process are you, the champion or *EIP author*, the [*EIP editors*](#eip-editors), and the [*Ethereum Core Developers*](https://github.com/ethereum/pm).
+Parties involved in the process are you, the champion or *EEP author*, the [*EEP editors*](#eep-editors), and the [*EOS Core Developers*]. [FLAG - WE WILL NEED TO MAKE A NEW HASHTAG TO REPLACE #eep-editors AND WE DON'T HAVE CORE DEVS]
 
-:warning: Before you begin, vet your idea, this will save you time. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [the Ethereum subreddit], [the Issues section of this repository], and [one of the Ethereum Gitter chat rooms]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EIP.
+:warning: Before you begin, vet your idea, this will save you time. Ask the EOS community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where EOS is used. Examples of appropriate public forums to gauge interest around your EEP include [the EOS subreddit], [the Issues section of this repository], and [SHOULD WE LIST A SITE LIKE EOS GO FORUMS?]. In particular, [FLAG - WE WOULD NEED TO SEE IF WE HAVE AN ISSUES SECTION OF THE REPO/WHAT WE END UP CALLING IT] [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EEP.
```

Comment:
> [SHOULD WE LIST A SITE LIKE EOS GO FORUMS?]

Yes IMO and probably also a few key Telegram channels.

> [FLAG - WE WOULD NEED TO SEE IF WE HAVE AN ISSUES SECTION OF THE REPO/WHAT WE END UP CALLING IT]

The GitHub repository that will holds the EEP will have a issues section: https://github.com/eoscanada/EEPs/issues